kubernetes/adapter_test.go

```diff
@@ -87,25 +95,18 @@ func TestMappingRoundtrip(tt *testing.T) {
 		},
 	} {
 		tt.Run(tc.msg, func(t *testing.T) {
-			got := newIngressFromKube(tc.kubeIngress)
+			a, _ := NewAdapter(testConfig, testIngressFilter, testSecurityGroup)
+
+			got := a.newIngressFromKube(tc.kubeIngress)
 			if !reflect.DeepEqual(tc.ingress, got) {
-				t.Errorf("mapping from kubernetes ingress to adapter failed. wanted %v, got %v", tc.ingress, got)
-			}
-			if got.CertificateARN != tc.kubeIngress.Metadata.Annotations[ingressCertificateARNAnnotation] {
-				t.Error("wrong value from CertificateARN()")
-			}
-			if got.Scheme != tc.kubeIngress.Metadata.Annotations[ingressSchemeAnnotation] {
-				t.Error("wrong value from Scheme()")
-			}
-			if got.Hostname != tc.kubeIngress.Status.LoadBalancer.Ingress[1].Hostname {
-				t.Error("wrong value from Hostname()")
+				t.Errorf("mapping from kubernetes ingress to adapter failed. wanted %#v, got %#v", tc.ingress, got)
```

Comment:
Maybe also change the message to use cmp.Diff(test.before, test.after) from https://godoc.org/github.com/google/go-cmp/cmp#example-Diff--Testing